integration_test/third_party_apps_data/applications/couchbase/metadata.yaml

```diff
@@ -0,0 +1,111 @@
+app_url: "https://www.couchbase.com/"
+short_name: couchbase
+long_name: Couchbase
+description: |-
+  The Couchbase integration collects bucket metrics such as operations, memory usage, and ejections. The integration collects metrics from the prometheus server exposed on a node.
+supported_app_version: ["6.5", "6.6", "7.0"]
+expected_logs:
+  - log_name: couchbase_general
+    fields:
+      - name: jsonPayload.message
+        type: string
+        description: "Log message"
+      - name: jsonPayload.node_name
+        type: string
+        description: The name of the node issuing the log message
+      - name: jsonPayload.module_name
+        type: string
+        description: The name of the module issuing the entry
+  - log_name: couchbase_http_access
+    fields:
+      - name: jsonPayload.message
+        type: string
+        description: "Log message"
+      - name: jsonPayload.client_ip
+        type: string
+        description: The ip address of the client invoking the HTTP request
+      - name: jsonPayload.user
+        type: string
+        description: The name of the user making the http request if basic auth is used.
+      - name: jsonPayload.method
+        type: string
+        description: The HTTP method of the request
+      - name: jsonPayload.status_code
+        type: integer
+        description: The status code of the response of the HTTP request
+  - log_name: couchbase_goxdcr
+    fields:
+      - name: jsonPayload.message
+        type: string
+        description: "Log message"
+      - name: log_type
+        type: string
+        description: The name of the component that is issuing the cross datacenter log
+expected_metrics:
+  - type: workload.googleapis.com/couchbase.bucket.operation.count
+    value_type: DOUBLE
+    kind: CUMULATIVE
+    monitored_resource: gce_instance
+    labels:
+      bucket_name: .*
+  - type: workload.googleapis.com/couchbase.bucket.item.count
+    value_type: DOUBLE
+    kind: CUMULATIVE
+    monitored_resource: gce_instance
+    labels:
+      bucket_name: .*
+  - type: workload.googleapis.com/couchbase.bucket.vbucket.count
+    value_type: DOUBLE
+    kind: CUMULATIVE
+    monitored_resource: gce_instance
+    labels:
+      bucket_name: .*
+  - type: workload.googleapis.com/couchbase.bucket.memory.usage
+    value_type: DOUBLE
+    kind: GAUGE
+    labels:
+      bucket_name: .*
+    monitored_resource: gce_instance
+    representative: true
+  - type: workload.googleapis.com/couchbase.bucket.item.ejection.count
+    value_type: DOUBLE
+    kind: GAUGE
+    monitored_resource: gce_instance
+    labels:
+      bucket_name: .*
+  - type: workload.googleapis.com/couchbase.bucket.error.oom.count
```

Comment:
For some reason, the test failed and this metric was not reported in the test project - do we know why?

Same thing for:

`workload.googleapis.com/couchbase.bucket.vbucket.count`
`workload.googleapis.com/couchbase.bucket.operation.count`
`workload.googleapis.com/couchbase.bucket.error.oom.count`

Reply:
Actively looking into these: working in my test environment 
<img width="1410" alt="image" src="https://user-images.githubusercontent.com/32067685/177816595-daf63fae-1d0c-4c1e-9087-8294377e7b02.png">

Re-running kokoro to see if I can find anything more